url_creator.py

```diff
@@ -0,0 +1,6 @@
+from os import path
+
+AMAZON_BASE_URL = 'https://www.amazon.com/dp'
```

Comment:
AMAZON_BASE_URL should be ''https://www.amazon.com' not 'https://www.amazon.com/dp', since the base URL should not contain product-related endpoint